doc/manual/manual.tex

```diff
@@ -6310,11 +6310,78 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 parameter (the division by 0.02 in the formula for the smoothed initial
 condition). However, for more exact results, one can choose
 this half width parameter proportional to the mesh size and thereby get more
-accurate results. Since the point of the section was to demonstrate the reason
-for the lack of convergence, but we leave this as an experiment to the reader.
+accurate results. The point of the section was to demonstrate the reason
+for the lack of convergence.
 
+In this section we extend the Van Keken cookbook following up the work previously completed by Cedric Thieulot, Juliane Dannberg,
+Timo Heister and Wolfgang Bangerth.  \textit{This section contributed by Grant Euen, Tahiry Rajaonarison, and Shangxin Liu as part of the Geodynamics and ASPECT class at Virginia Tech.}  
 
+As already mentioned above, using a half width parameter proportional to the mesh size allows for more accurate results.  We test the effect of the half width size of the smoothed discontinuity by changing the division by 0.02, the smoothing parameter, in the formula for the smoothed initial conditions into values proportional to the mesh size.  We use global refinement degree 7 because the root mean square velocity converges at greater refinement degree while keeping average runtime around 5 to 25 minutes.  These runtimes were produced by the BlueRidge cluster of the Advanced Research Computing (ARC) program at Virginia Tech.  BlueRidge is a 408-node Cray CS-300 cluster; each node outfitted with two octa-core Intel Sandy Bridge CPUs and 64 GB of memory.  A chart of average runtimes for global mesh refinements 5 through 10 on one node can be seen in Table~\ref{tab:runtime-table}.  For global refinement degree 7 (128$\times$128 mesh size), the size of the mesh is 0.0078 corresponding to a half width parameter of 0.0039.  The smooth model allows for much better convergence of the secondary plumes, although they are still more scattered than the primary plumes.
 
+\begin{table}[htb]
+	\center
+	\begin{tabular}{|c|cccccc|}
+		\hline
+		Global & & Number of Processors & & \\
+		Refinements & 4 & 8 & 12 & 16
```

Comment:
In the first line of the table heading here, you want "Number of processors" to span all of "4 & 8 & 12 & 16". You can achieve this by telling latex that a text spans multiple column:
```
  Global & \multicolumn{4}{|c|}{Number of processors}
```